extensions-core/namespace-lookup/src/main/java/io/druid/query/lookup/namespace/StaticMapExtractionNamespace.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.lookup.namespace;
+
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.google.common.base.Preconditions;
+
+import java.util.Map;
+
+@JsonTypeName(StaticMapExtractionNamespace.TYPE_NAME)
+public class StaticMapExtractionNamespace implements ExtractionNamespace
+{
+  static final String TYPE_NAME = "staticMap";
+  private final Map<String, String> map;
+
+  @JsonCreator
+  public StaticMapExtractionNamespace(
```

Comment:
I had already done that but forgot to push: https://github.com/druid-io/druid/pull/2926/commits/365d8f1f3200bcd742b133e2205cdb1695ee12ea